src/mmw/js/src/analyze/templates/catchmentWaterQualityTable.html

```diff
@@ -3,25 +3,23 @@
       <thead>
           <tr>
               <th data-sortable="true">Id</th>
-              <th data-sortable="true" data-sorter="window.noDataSort">
-                  Area (ha)
-              </th>
-              <th class="text-right" data-sortable="true" data-sorter="window.noDataSort">
+              <th data-sortable="true" data-field="areaha"> Area (ha)</th>
```

Comment:
Should this maintain the `data-sorter` attribute?  Curious about the relationship between this and this recent commit: https://github.com/WikiWatershed/model-my-watershed/pull/1591/commits/0a7a5446db7768956954b8bfef9241cec27d0005